src/poetry/puzzle/provider.py

```diff
@@ -666,23 +666,23 @@ def complete_package(
         )
 
         # Searching for duplicate dependencies
-        #
+
         # If the duplicate dependencies have the same constraint,
         # the requirements will be merged.
-        #
+
         # For instance:
-        #   - enum34; python_version=="2.7"
-        #   - enum34; python_version=="3.3"
-        #
+        #   - enum34; python_version=="2.7"  # noqa: ERA001
```

Comment:
```suggestion
        #   • enum34; python_version=="2.7"
```

What about using a unicode bullet? I think I prefer  this.

Reply:
Good idea. Replaced in comments across whole file for consistency